src/main/java/org/opentripplanner/gtfs/mapping/GTFSToOtpTransitServiceMapper.java

```diff
@@ -72,7 +76,16 @@
         agencyMapper = new AgencyMapper(feedId);
         routeMapper = new RouteMapper(agencyMapper);
         tripMapper = new TripMapper(routeMapper);
-        stopTimeMapper = new StopTimeMapper(stopMapper, locationMapper, locationGroupMapper, tripMapper);
+
+        agencyAndIdMapper = new AgencyAndIdMapper();
+        bookingRuleMapper = new BookingRuleMapper(agencyAndIdMapper);
+        stopTimeMapper = new StopTimeMapper(
```

Comment:
After extending the `StopTimeMapper` with `bookingRuleMapper` as an input, I started to get compilation error: 
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/BookingRuleMapper.java:[16,13] cannot find symbol
  symbol: variable agencyAndIdMapper
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/BookingRuleMapper.java:[33,19] cannot find symbol
  symbol:   variable agencyAndIdMapper
  location: class org.opentripplanner.gtfs.mapping.BookingRuleMapper
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/BookingRuleMapper.java:[41,37] cannot find symbol
  symbol:   variable agencyAndIdMapper
  location: class org.opentripplanner.gtfs.mapping.BookingRuleMapper
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/StopTimeMapper.java:[38,13] cannot find symbol
  symbol: variable bookingRuleMapper
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/StopTimeMapper.java:[75,34] cannot find symbol
  symbol:   variable bookingRuleMapper
  location: class org.opentripplanner.gtfs.mapping.StopTimeMapper
[ERROR] /Users/victor/kyyticom/OpenTripPlanner/src/main/java/org/opentripplanner/gtfs/mapping/StopTimeMapper.java:[76,35] cannot find symbol
  symbol:   variable bookingRuleMapper
  location: class org.opentripplanner.gtfs.mapping.StopTimeMapper
```
I believe the real root cause is in the way how `agencyAndIdMapper` is initialized, but I could not figure out how to fix it. 😞

Reply:
With latest code changes, the code compiles successfully